src/commands/registerBotCommands.js

```diff
@@ -34,7 +35,11 @@ export default async bot => {
 		{ command: 'start', handler: startHandler, description: 'Запустить бота' },
 		{ command: 'week', handler: getWeekInfoHandler, description: 'Какая сейчас неделя' },
 		{ command: 'week_schedule', handler: getWeekScheduleHandler({ weekOffset: false }), description: 'Расписание на эту неделю' },
-		{ command: 'next_week_schedule', handler: getWeekScheduleHandler({ weekOffset: true }), description: 'Расписание на неделю' },
+		{
+			command: 'next_week_schedule',
+			handler: getWeekScheduleHandler({ weekOffset: true }),
```

Comment:
I can't think of a better name, since `week offset' is a direct translation of what I mean. Perhaps you can tell me how best to proceed?

Reply:
Ok, you have `getWeekScheduleHandler` method. What would you expect it to do?

I suppose it should return some kind of schedule (which is already kinda working) and have some configuration. As we don't directly mention from which week it should get the schedule it would be reasonalble to expect it to take the **week number** as parameter.

You already have the method to determine the current week number by current day - so you can just remove this boolean params and pass `weekNumber` everywhere.

Reply:
Ok, I made changes to the code. Yes, you are right, this is much better